cli/src/semgrep/app/scans.py

```diff
@@ -152,6 +152,8 @@ def fetch_config_and_start_scan_old(self, meta: Dict[str, Any]) -> None:
         self._skipped_syntactic_ids = body.get("triage_ignored_syntactic_ids") or []
         self._skipped_match_based_ids = body.get("triage_ignored_match_based_ids") or []
 
+        state.error_handler.append_request(scan_id=self.scan_id)
```

Comment:
what happens here in the case that it doesn't go far enough to get a scan_id?

Reply:
If the scan was not created successfully, the CLI would not get to this line and would not be included in the fail-open POST. The intention here is to include the `scan_id` IFF the scan was successfully created on the server.

Reply:
check, sounds good!